server/routes/records.js

```diff
@@ -6,34 +6,16 @@ var fs = require("fs");
 //const { StringDecoder } = require("string_decoder");
 //const decoder = new StringDecoder("utf8");
 
-function jsonWrite(name, mJson) {
-    fs.writeFile(name, mJson, function(err) {
-        if (err) {
-            console.log(err);
-        }
-        console.log("complete");
-    });
-}
-
 router.post("/", async(req, res) => {
     //  console.log("1")
     console.log(req.body)
 
+    fs.writeFile("test.json", JSON.stringify(req.body, null, 2), () => {
+        res.header({
+            "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
Forgot to fix this to `application/json`. This may not even be needed - I'm not sure but `res.json` may do this for you automatically.